rasa/core/tracker_store.py

```diff
@@ -161,18 +162,18 @@ def create(
         import pymongo.errors
         import sqlalchemy.exc
 
+        if isinstance(obj, EndpointConfig):
+            result = plugin_manager().hook.update_endpoint_config(endpoint_config=obj)
+            obj = result[0] if result else obj
```

Comment:
if we can only handle on hook impl, we should use https://pluggy.readthedocs.io/en/stable/#first-result-only